sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/TerrainStrategies/ElevatedTerrainStrategy.cs

```diff
@@ -44,9 +44,15 @@ public override void RegisterTile(UnityTile tile)
 				tile.gameObject.layer = _elevationOptions.unityLayerOptions.layerId;
 			}
 
-			if (tile.MeshFilter.mesh.vertexCount == 0)
+			//_newVertexList.Count is the vertex count this strategy is expected to use
```

Comment:
Is this comment relevant? the check is changed to a different thing.

Reply:
@brnkhy

Reply:
no not anymore, that check will probably change soon as well